docs/_blog/introducing-rapid-gossip-sync.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Introducing Rapid Gossip Sync"
+description: "Rapid Gossip Sync is a protocol that allows nodes to catch up with channel gossip data by relying on semi-trusted servers."
+date: "2022-08-30"
+authors:
+  - Arik Sosman
+tags:
+  - Gossip 
+  - Network Graph 
+--- 
+
+If you've run a Lightning node, you'll have noticed that initialization can be time-consuming. Merely opening a channel requires six block confirmations or about one hour. However, even if your node already has open channels but has been offline for a while, restarting it still requires fetching the latest channel graph data (commonly referred to as "gossip”) and verifying that all its channels have remained intact.
+
+While most Lightning implementations have mechanisms to quickly catch up to the latest block, one bottleneck that may remain is the exchange of aforementioned gossip, which rarely takes more than a couple of minutes but significantly impacts UX when initiating locally-routed payments from a mobile device. Assuming that an app remains closed until you need to send a payment, even a one-minute waiting period becomes a major payment flow disruptor. This issue is commonly resolved by sharing the intended payment recipient with a Lightning Service Provider, thus compromising privacy to offload the route calculation to a server.
+
+For that reason, we're very excited to announce Rapid Gossip Sync, a protocol that allows nodes to catch up with channel gossip data by relying on semi-trusted servers. 
+
+## Exchanging Gossip Today
+
+Before we delve too deeply into the optimizations that Rapid Gossip Sync relies on, first a primer on how gossip works today. It involves three main message types: channel announcements, channel updates, and node announcements. (The last of which isn’t really relevant to this post.)
+
+As their names imply, channel announcements are sent when a public channel opens, and updates when changes are made to a channel. One peculiar characteristic of channels is that they aren't symmetrical: depending on the direction, some of their properties may vary. For instance, if Alice and Bob have a channel, Alice may charge higher fees for forwarding payments to Bob than vice versa, thus requiring that different channel updates be broadcast to the network for each direction. Just as balances may concentrate on one side, routing fees for forwarding payments through the channel are set by different parties depending on the direction. Unlike channel updates, channel announcements do not carry directional information, meaning opening a new channel requires sending at least two channel updates for direction-specific data.
```

Comment:
thus requiring that different channel updates be broadcast to the network for each direction, **because channel announcements cannot contain directional data**.

Reply:
let me edit that on Google. I am just now noticing that this information is conveyed a bit late in that paragraph.